src/chapters/exceptions/introduction.rst

```diff
@@ -0,0 +1,161 @@
+.. index:: ! exception, ! throw
+
+Exceptions
+==========
+
+Like most programming languages, C# includes exceptions and exception handling tooling. Some exception objects
+are provided by .NET itself, while others we may encounter from external C# libraries, or write ourselves.
+
+**Exceptions** in C# are objects derived from the ``System.Exception`` class. They are used in circumstances where something
+takes place outside of the ordinary flow of a program. When you want to explicitly call an exception in your code, you do so 
```

Comment:
What about "Exceptions occur at runtime when something takes place outside of the expected flow of our code."